python/gps/gps_main.py

```diff
@@ -69,10 +69,14 @@ def run(self, itr_load=None):
                 self.agent.get_samples(cond, -self._hyperparams['num_samples'])
                 for cond in self._train_idx
             ]
+
             self._take_iteration(itr, traj_sample_lists)
             pol_sample_lists = self._take_policy_samples()
             self._log_data(itr, traj_sample_lists, pol_sample_lists)
 
+            # Clear agent samples to prevent memory leak.
```

Comment:
nitpic: maybe just "Clear agent samples". It's not exactly a memory leak, more like excessive memory usage.